committeeoversightapp/views.py

```diff
@@ -92,7 +92,7 @@ def filter_queryset(self, qs):
         if detail_type == 'category':
             qs = qs.filter(hearingcategory__category_id=id)
         if detail_type == 'committee':
-            pass
+            qs = qs.filter(participants__organization_id=id)
```

Comment:
We'll want hearings from a committee and also its subcommittees, so something like this:

```python
from django.db.models import Q

qs = qs.filter(Q(participants__organization_id=id) | Q(participants__parent_id=id))
```